website/data/language-nav-data.json

```diff
@@ -763,8 +767,13 @@
         "hidden": true
       },
       {
-        "title": "base64sha256",
-        "path": "language/functions/base64sha256",
+        "title": "base64gzip",
```

Comment:
It looks like you accidentally added an additional base64gzip entry and removed the base64sha256 entry here.

Reply:
It also looks like this list is alphabetical and your entry for gunzipbase64 should be lower down.

Reply:
@l-with sorry to nit-pick, but base64sha256 should come after base64gzip.  Once that is addressed, this is ready to merge.